src/main/java/world/bentobox/bentobox/Settings.java

```diff
@@ -15,772 +13,900 @@
  * All the plugin settings are here
  * @author tastybento
  */
-@StoreAt(filename="config.yml") // Explicitly call out what name this should have.
+@StoreAt(filename = "config.yml") // Explicitly call out what name this should have.
 @ConfigComment("BentoBox v[version] configuration file.")
 @ConfigComment("")
-@ConfigComment("This configuration file contains settings that mainly apply to or manage the following elements:")
+@ConfigComment(
+  "This configuration file contains settings that mainly apply to or manage the following elements:"
+)
```

Comment:
Why?

Reply:
90 char per line. It does not know how to split text that is a parameter.